package.json

```diff
@@ -24,6 +24,8 @@
     "codemirror": ">=5.10 <6.0",
     "jquery": ">=2.2 <2.3",
     "jshint": "^2.9.5",
+    "less": "^2.7.3",
+    "phantomjs-prebuilt": "^2.1.16",
```

Comment:
What changed that required these be added now, if they were already used before?

Reply:
All I know is that it didn't work without them. Maybe previous testers have had less and phantomjs installed globally and I don't?